src/Linux/Core/Impl/IO/UnixFileSystem.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Common.Core;
+using Microsoft.Common.Core.IO;
+using Microsoft.Extensions.FileSystemGlobbing;
+
+namespace Microsoft.R.Common.Core.Linux {
```

Comment:
I believe we have decided to use one common namespace for both xplat types and platform-specific types. So it should be `Microsoft.Common.Core.IO`.